src/createLoadableVisibilityComponent.js

```diff
@@ -1,8 +1,8 @@
-import React, { Component } from 'react'
+import * as React from 'react'
```

Comment:
Would you be able to revert this change? I'd prefer to be explicit about imports.